python/ml4ir/applications/ranking/model/layers/masking.py

```diff
@@ -133,6 +135,10 @@ class QueryFeatureMask(layers.Layer):
                 [0., 1., 1., 1.]]])
     """
 
+    masking_config = {MonteCarloInferenceKey.FIXED_MASK_COUNT: 1,
+                    MonteCarloInferenceKey.USE_FIXED_MASK_IN_TRAINING: False,
+                    MonteCarloInferenceKey.USE_FIXED_MASK_IN_TESTING: False}
+
```

Comment:
@mohazahran Why not make these arguments to this constructor when initializing?

Reply:
I am a little confused that these arguments are hardcoded. How does one override them? Wouldn't it be easier to read these as arguments or from model config?

Reply:
So you are using python globals instead of tensorflow globals? Did you check if this works with the sanity tests in python and java? We might have to revisit with tf globals otherwise.